spriint7/git/tema3.js

```diff
@@ -0,0 +1,39 @@
+//cream o functie cu zilele saptmani iar ca si parametru va fi numarul zilei
+function zilele_saptamani(nrZile){
```

Comment:
la fel ca si in tema1, hai sa alegem un nume mai potrivit.
Ce face aceasta functie? Returneaza textul saptamanii. Si ce primeste ca argument? Numarul saptamanii.
Cum ar fi sa o numim asa:

```
function get_ziua_ca_si_text(numarulZileiDinSaptamana)
```

Daca iti pare numele numarulZileiDinSaptamana este prea lung - ai putea folosi un simplu "n" sau altceva scurt. nrZile e ok si el, dar pe mine ma duce in confuzie.